homotopy-web/src/app/boundary.rs

```diff
@@ -78,6 +100,7 @@ impl BoundaryPreview {
                     diagram={ctx.props().boundary.diagram.clone()}
                     id="boundary__preview"
                     signature={ctx.props().signature.clone()}
+                    dimensions={Some((160.0, 160.0))}
```

Comment:
Can you please rename this to `max_dimensions`? Or maybe split it into `max_width` and `max_height`?

Reply:
Sure, just committed.

Reply:
Thanks!